### ISSUE TITLE
RxJS 6 support

### ISSUE THREAD

**Issue by Koslun** [bleenco/ngx-uploader#439]
ngx-uploader version 4.24
angular version 6.0.0
rxjs version 6.1.0
typescript version 2.7.2
Node.js version 8.9.1
yarn version 1.5.1

**Expected**
Angular 6 default configuration should work, including RxJS 6.

**Actual**
Angular 6 default configuration only works only when also using `rxjs-compat`.

Without the `rxjs-compat` I get this error.
```
ERROR in node_modules/ngx-uploader/dist/src/ngx-uploader/classes/interfaces.d.ts(1,10): error TS2305: Module '"C:/Users/adamw/repos/opensource/ng-example-app/node_modules/rxjs/Subscription"' has no exported member 'Subscription'.
node_modules/ngx-uploader/dist/src/ngx-uploader/classes/ngx-uploader.class.d.ts(2,10): error TS2305: Module '"C:/Users/adamw/repos/opensource/ng-example-app/node_modules/rxjs/Observable"' has no exported member 'Observable'.
node_modules/ngx-uploader/dist/src/ngx-uploader/classes/ngx-uploader.class.d.ts(3,10): error TS2305: Module '"C:/Users/adamw/repos/opensource/ng-example-app/node_modules/rxjs/Subject"' has no exported member 'Subject'.
node_modules/ngx-uploader/dist/src/ngx-uploader/classes/ngx-uploader.class.d.ts(4,10): error TS2305: Module '"C:/Users/adamw/repos/opensource/ng-example-app/node_modules/rxjs/Subscription"' has no exported member 'Subscription'.
node_modules/ngx-uploader/dist/src/ngx-uploader/directives/ng-file-drop.directive.d.ts(3,10): error TS2305: Module '"C:/Users/adamw/repos/opensource/ng-example-app/node_modules/rxjs/Subscription"' has no exported member 'Subscription'.
node_modules/ngx-uploader/dist/src/ngx-uploader/directives/ng-file-select.directive.d.ts(3,10): error TS2305: Module '"C:/Users/adamw/repos/opensource/ng-example-app/node_modules/rxjs/Subscription"' has no exported member 'Subscription'.
node_modules/rxjs/Observable.d.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/Observable'.
node_modules/rxjs/Subject.d.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/Subject'.
node_modules/rxjs/Subscription.d.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/Subscription'.
```

**Reproduction**
[Here ](https://github.com/Koslun/ng-example-app/tree/ng6-uploader)is an Angular 6 app generated with default angular-cli version 6 settings with an ngx-uploader exampled added to it. The latest commit produces above error, commit before including `rxjs-compat` is.

**Comment by jkuri** [comment]
thanks for report. I am accepting PRs.

**Comment by Koslun** [comment]
Solved in PR #440
